### ISSUE TITLE
Problems with dark theme color scheme   Deadbeef _1.9.6

### ISSUE THREAD

**Issue by sbfwfnh** [DeaDBeeF-Player/deadbeef#3121]
First of all, thank you all for your selfless dedication to provide such a banging music player for linux!

Here is my feedback on the problem:
Dark system theme in mode, play and pause buttons are not visible at all

Deadbeef version: 1.9.6
OS: Kubuntu_224.04 deepin_V23_linux

![deadbeef_1 9 6](https://github.com/user-attachments/assets/5436f6bc-69e3-4b92-838e-839ab47490d9)

**Comment by Oleksiy-Yakovenko** [comment]
Are you running deadbeef with gtk2 or gtk3 ui? Looks like some sort of gtk icon theme mismatch. Any chance that you could try checking if any other apps have similar issues?

**Comment by sbfwfnh** [comment]
Yes, I use GTK3 as the player, and I haven't noticed this issue with other software yet!

**Comment by sbfwfnh** [comment]
Additionally, I am using a lyric plugin ：ddb_lyricbar_gtk3 
dlopen error: /opt/deadbeef/lib/deadbeef/dca.so: undefined symbol: dca_dynrng
 
I am a beginner and not very familiar with it. I hope you can help me clarify my doubts. Thank you.

**Comment by Oleksiy-Yakovenko** [comment]
I can only suggest to try changing GTK icon theme to some other one. Can't see what deadbeef could be doing wrong there. It usually works. But I'll try reproducing this when I get the chance.

**Comment by Oleksiy-Yakovenko** [comment]
Closing, not an issue with deadbeef. Find a way to change your icon theme.
Bug #3170 looks similar, and reported somehow managed to resolve it locally.
